execute-generators/src/main/kotlin/de/itemis/mps/gradle/generate/Helper.kt

```diff
@@ -56,15 +57,17 @@ private fun createScript(proj: Project, models: List<org.jetbrains.mps.openapi.m
     val scb = ScriptBuilder()
 
     scb.withFacetNames(allUsedLanguages
-            .mapNotNull { registry.getLanguage(it) }
-            .mapNotNull { it.getAspect(MakeAspectDescriptor::class.java) }
-            .flatMap { it.manifest.facets() }
-            .map { it.name }
+            ?.mapNotNull { registry.getLanguage(it) }
```

Comment:
I'd prefer a message here if allUsedLanguages turn out to be null.
Looks like the proposed version would silently do nothing in this case?

Reply:
Should resolve with https://github.com/mbeddr/mps-gradle-plugin/pull/36/commits/48c1c05207fec32b41c1d5ce75ebde1e99926f62